README.md

```diff
@@ -6,189 +6,95 @@
-new [cisagov](https://github.com/cisagov) GitHub Docker project
-started.  This skeleton project contains [licensing
-information](LICENSE), as well as [pre-commit hooks](https://pre-commit.com)
-and [GitHub Actions](https://github.com/features/actions) configurations
-appropriate for Docker containers and the major languages that we use.
+This Docker project serves as the vessel for certificate transparency
+scanning performed by the [admiral Python library](https://github.com/cisagov/admiral).
 
 ## Running ##
 
-### Running with Docker ###
-
-To run the `cisagov/example` image via Docker:
-
-```console
-docker run cisagov/example:0.0.1
-```
-
 ### Running with Docker Compose ###
 
-1. Create a `docker-compose.yml` file similar to the one below to use [Docker Compose](https://docs.docker.com/compose/).
-
-    ```yaml
-    ---
-    version: "3.7"
-
-    services:
-      example:
-        image: cisagov/example:0.0.1
-        volumes:
-          - type: bind
-            source: <your_log_dir>
-            target: /var/log
-        environment:
-          - ECHO_MESSAGE="Hello from docker compose"
-        ports:
-          - target: 8080
-            published: 8080
-            protocol: tcp
-    ```
-
+1. Change the credentials in `secrets`
+1. Choose configuration options for `admiral.yml`
 1. Start the container and detach:
 
     ```console
     docker compose up --detach
     ```
 
-## Using secrets with your container ##
-
-This container also supports passing sensitive values via [Docker
-secrets](https://docs.docker.com/engine/swarm/secrets/).  Passing sensitive
-values like your credentials can be more secure using secrets than using
-environment variables.  See the
-[secrets](#secrets) section below for a table of all supported secret files.
-
-1. To use secrets, create a `quote.txt` file containing the values you want set:
-
-    ```text
-    Better lock it in your pocket.
-    ```
-
-1. Then add the secret to your `docker-compose.yml` file:
-
-    ```yaml
-    ---
-    version: "3.7"
-
-    secrets:
-      quote_txt:
-        file: quote.txt
-
-    services:
-      example:
-        image: cisagov/example:0.0.1
-        volumes:
-          - type: bind
-            source: <your_log_dir>
-            target: /var/log
-        environment:
-          - ECHO_MESSAGE="Hello from docker compose"
-        ports:
-          - target: 8080
-            published: 8080
-            protocol: tcp
-        secrets:
-          - source: quote_txt
-            target: quote.txt
-    ```
-
-## Updating your container ##
-
-### Docker Compose ###
-
-1. Pull the new image from Docker Hub:
-
-    ```console
-    docker compose pull
-    ```
-
-1. Recreate the running container by following the [previous instructions](#running-with-docker-compose):
-
-    ```console
-    docker compose up --detach
-    ```
-
-### Docker ###
-
-1. Stop the running container:
-
-    ```console
-    docker stop <container_id>
-    ```
-
-1. Pull the new image:
```

Comment:
Why did you remove the "Using secrets with your container" and "Updating your container" sections?  Those both seem like they would be useful to include in the README.

Reply:
Good point! I was looking more closely at the Admiral [README](https://github.com/cisagov/admiral/README.md) when I put this one together. But I agree. I dropped the commit where I removed these sections.

Reply:
Thanks for doing that.  Next, you should customize the "Using secrets with your container" section to make it applicable and useful for this repo.

Reply:
I edited [174d98b](https://github.com/cisagov/admiral-docker/pull/5/commits/174d98b96e1d5378017059ce32360fc871f7be88) to be more specific about how secrets are used in this repository. I thought the steps for creating a secret and adding it to the composition were irrelevant, but I can add them back in if you think they're useful.

Reply:
What you have in there now works for me - thanks!